### ISSUE TITLE
add point_cauchy prior family

### ISSUE THREAD

**Issue by willwerscheid** [stephenslab/ebnm#51]
interface to EbayesThresh

**Comment by stephens999** [comment]
does EbayesThresh do point-cauchy? I thought it was either point-laplace or cauchy? maybe i'm misrembering

**Comment by willwerscheid** [comment]
yes, it's point-Cauchy (or rather, a mixture of a pointmass and a scale mixture of normals with polynomial tails that the authors call "quasi-Cauchy")

**Comment by pcarbo** [comment]
There are two options: `prior = "cauchy"` and `prior = "laplace"`. Only the laplace prior is allowed for case of heterogeneous variances.

**Comment by willwerscheid** [comment]
right, but the "laplace" option is actually point-laplace and "cauchy" is as i've described above

**Comment by pcarbo** [comment]
Yes, you are right to clarify that point.

**Comment by stephens999** [comment]
thanks for clarification - yes, i was wrong

**Comment by willwerscheid** [comment]
EBayesThresh doesn't return a posterior second moment or sampler, so adding the point-Cauchy family might be more trouble than it's worth
